index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <h1 id="title">Our <strong>Survey</strong> Form</h1>
```

Comment:
It would be better to use semantic tags. Instead of using the `<div>` tag for the title, you can use the `<header>`. And in the body, you can use the `<section>`  for the contents instead of `div`.

Reply:
- [ ] Kindly add an explanation -`WHY`- for your change request, I think it will be better if you include something like: "Respecting HTML semantic tags can make your code much easier to read, and it has greater accessibility". I believe that you can add some links to make your review as an oppurtunity for the student to learn -->  [HTML Semantic Elements
](https://www.w3schools.com/html/html5_semantic_elements.asp)